xla/service/gpu/gpu_windowed_einsum_handler_test.cc

```diff
@@ -44,6 +44,16 @@ HloInstruction* FindInstructionByName(HloComputation* comp, std::string name) {
   return nullptr;
 }
 
+std::vector<HloInstruction*> GatherAllInstructions(const HloComputation& computation, HloOpcode opcode) {
```

Comment:
Can we reuse this from or move it to `utils/hlo_query.h`?

Reply:
this is removed in favor of using filecheck